packages/cojson-transport-ws/src/index.ts

```diff
@@ -5,30 +5,27 @@ import {
     SyncMessage,
     cojsonInternals,
 } from "cojson";
-import { AnyWebSocket, PingMsg } from "./types.js";
-
- const g: typeof globalThis & {
-    jazzPings?: {
-        received: number;
-        sent: number;
-        dc: string;
-    }[];
-} = globalThis;
+import { AnyWebSocket } from "./types.js";
+import { BatchedOutgoingMessages } from "./BatchedOutgoingMessages.js";
 
 export const BUFFER_LIMIT = 100_000;
 export const BUFFER_LIMIT_POLLING_INTERVAL = 10;
 
+export type CreateWebSocketPeerOpts = {
+    id: string;
+    websocket: AnyWebSocket;
+    role: Peer["role"];
+    expectPings?: boolean;
+    batchingByDefault?: boolean;
+};
+
 export function createWebSocketPeer({
     id,
     websocket,
     role,
     expectPings = true,
-}: {
-    id: string;
-    websocket: AnyWebSocket;
-    role: Peer["role"];
-    expectPings?: boolean;
-}): Peer {
+    batchingByDefault = false, // Keeping this false until we release batching upgrade to the mesh
```

Comment:
I think that's better to keep it disabled until the version of `createWebSocketPeer` with batching is released to the production mesh.

This way we can first upgrade the mesh to support batching and then enable batching to everyone.

Since the pets e2e tests now run with the local sync we can ensure that the server side supports client that aren't batching.

Reply:
Perfect. Thanks for thinking about how to roll this out